atom/common/deprecate_util.cc

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2019 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include <string>
```

Comment:
_(nit)_ string is part of the public API so it's already included in the header, no need to re-include here